obspy/taup/tau.py

```diff
@@ -20,6 +21,8 @@
 from .taup_pierce import TauP_Pierce
 from .taup_time import TauP_Time
 
```

Comment:
Sorry, this is not the empty line you should keep. There should be two after the the imports.